configs/rodent.yaml

```diff
@@ -88,9 +90,13 @@ KEYPOINT_INITIAL_OFFSETS:
   WristR: 0. 0. 0.0
 
 TRUNK_OPTIMIZATION_KEYPOINTS:
-  - "Spine"
-  - "Hip"
-  - "Shoulder"
+  - "SpineF"
+  - "SpineL"
+  - "SpineM"
+  - "HipL"
+  - "HipR"
+  - "ShoulderL"
+  - "ShoulderR"
```

Comment:
How are these changes related to the scope of this PR? I don't see them mentioned in the description.

Reply:
The previous keypoint names were not specific `Spine` versus `SpineF`, and [the logic](https://github.com/talmolab/stac-mjx/blob/04568a1c6a50bbeb305288e6a5d93a0cc58859e8/stac_mjx/compute_stac.py#L46-L51) to get the correct corresponding sites were accordingly fuzzy: 
[I updated the logic](https://github.com/talmolab/stac-mjx/pull/43/files/e246ff026ebfd8840307a9d29a7ce4343bffc9e1#diff-a13ae84d216cd39bfbdae18a8b4ee9d8c4bcd61faf116f3db7e9c538ca013802R57-R59) to be more precise